Intersect (Core)/CustomColors.cs

```diff
@@ -82,6 +82,9 @@ public sealed partial class NamesNamespace
             public Dictionary<string, LabelColor> Players = new Dictionary<string, LabelColor>()
             {
                 {"Normal", new LabelColor(new Color(255, 255, 255, 255), Color.Black, new Color(180, 0, 0, 0))},
+                {"PartyMember", new LabelColor(new Color(255, 70, 192, 219), Color.Black, new Color(180, 0, 0, 0))},
+                {"GuildMember", new LabelColor(new Color(255, 5, 179, 26), Color.Black, new Color(180, 0, 0, 0))},
```

Comment:
can you change these to just Party and Guild

Reply:
reason: I want to take advantage of `nameof()` wherever possible in the future and I don't see us ever creating types called `PartyMember` and `GuildMember`

Reply:
Good catch, fixed